src/main/java/com/teragrep/k8s_01/config/AppConfig.java

```diff
@@ -42,4 +43,11 @@ public AppConfigRelp getRelp() {
     public String toString() {
         return new Gson().toJson(this);
     }
+
+    @Override
+    public void validate() throws InvalidConfigurationException {
```

Comment:
this does not verify that the actual elements are not null i.e. AppConfigKubernetes missing completely